build_common.ps1

```diff
@@ -896,6 +896,28 @@ class PassthroughReceiver : StdoutReceiver {
 	}
 }
 
+class BufferingReceiver : StdoutReceiver {
+	[object] $logLines
+	BufferingReceiver(){
+		$this.logLines = New-Object System.Collections.Generic.List[System.Object]
```

Comment:
An (incredibly small) optimization would be pre-sizing the list since we know the (rough) number of output lines that will be printed (563, bump up to 570 or something for a bit of room for any variation) by [passing the capacity to the constructor](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.-ctor?view=net-5.0#System_Collections_Generic_List_1__ctor_System_Int32_) (does ps support overloaded constructors?) or by [setting a property](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.capacity?view=net-5.0#System_Collections_Generic_List_1_Capacity)